WebDriverAgentLib/Categories/XCUIElement+FBUtilities.m

```diff
@@ -191,11 +222,25 @@ - (NSArray *)fb_createAXAttributes: (BOOL)asNumber
   return propertyNames;
 }
 
+/**
+ Returns root element query either with includingNonModalElements or no includingNonModalElements
+
+ @return XCUIElementQuery with includingNonModalElements or not
+*/
+- (XCUIElementQuery *)fb_rootXCUIElementQuery
```

Comment:
this helper does not return the root query, but just a query with non modal element included